cmd/jaeger/internal/integration/README.md

```diff
@@ -63,6 +63,56 @@ flowchart LR
     end
 ```
 
+## Kafka Integration
+
+The primary difference between the Kafka integration tests and other integration tests lies in the flow of data. In the standard tests, spans are written by the SpanWriter, sent through an RPC_client directly to a receiver, then to an exporter, and stored in an in-memory backend. Spans are read by the SpanReader, which queries the jaeger_query process accessing the storage backend. In contrast, the Kafka tests introduce Kafka as an intermediary. Spans go from the SpanWriter through an RPC_client to an OTLP receiver in the Jaeger Collector, exported to Kafka, received by the Jaeger Ingester, and then stored. For details, see the [Architecture](#KafkaArchitecture) section below.
+
+
+## Kafka Architecture
+
+``` mermaid
+flowchart LR
+    Test -->|writeSpan| SpanWriter
+    SpanWriter --> RPCW[RPC_client]
+    RPCW --> OTLP_Receiver[Receiver]
+    OTLP_Receiver --> CollectorExporter[Kafka Exporter]
+    CollectorExporter --> Kafka[Kafka]
+    Kafka --> IngesterReceiver[Kafka Receiver]
+    IngesterReceiver --> IngesterExporter[Exporter]
+    IngesterExporter --> StorageBackend[(In-Memory Store)]
+    Test -->|readSpan| SpanReader
+    SpanReader --> RPCR[RPC_client]
+    RPCR --> QueryProcess[jaeger_query]
+    QueryProcess --> StorageBackend
+    IngesterExporter -.-> StorageCleaner
```

Comment:
what does this edge represent? I don't think exporter communicates to cleaner in any way.

Reply:
![image](https://github.com/user-attachments/assets/c70087cb-02f7-4179-83ef-3e145c74c0c5)
The dotted edges represent this section ( yes, there is no communication between the exporter and the cleaner )

Reply:
Fixed it in the latest fix